angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
+		<p>{{ welcome.accessibilityText }}</p>
+	</div>
+  <!-- <p class="visible-xs-* learn-more">Learn more at the <a href="#">KnowledgeBase.</a></p> -->
```

Comment:
That could be interesting. But the KB article would still only have to be added to one file, this one, currently. So it's probably more work to write that logic than to simply uncomment and stick in the links here.